### ISSUE TITLE
Change R Inferior buffer name in Windows User's Home Directory 

### ISSUE THREAD

**Issue by ChandlerLutz** [emacs-ess/ESS#1073]
In Windows, 

If I start a new R inferior ess session in the user home directory, the directory gets appended to the buffer's name. 

So, if I start a inferior R process in `~/.emacs.d/` the name of the iESS buffer will be `*R:.emacs.d` 

I believe that this is new behavior

Outside of the home directory, the inferior buffer is simply *R* no matter what the directory is. 

Question: Inside the `~` directory, how can I set the name of the iESS buffer to just `*R*`. 

For reference, I'm using this rather old code that allows shift-enter to start a new buffer, etc. 

https://www.r-bloggers.com/2009/10/make-shift-enter-do-a-lot-in-ess/

**Comment by jabranham** [comment]
What is the value of ess-gen-proc-buffer-name-function and does setting
it to ess-gen-proc-buffer-name:simple help?

**Comment by ChandlerLutz** [comment]
Thanks! 

The value of `ess-gen-proc-buffer-name-function` was set to `ess-gen-proc-buffer-name:project-or-simple`

In my init file I used 

`(setq ess-gen-proc-buffer-name-function 'ess-gen-proc-buffer-name:simple)` 

and this solved the issue. 

You are probably already aware (in case someone else comes across this), but to see the value of `ess-gen-proc-buffer-name-function`, I had to evaluate it with `M-:` and then paste it in. It wasn't available in help searches via `C-h f` or C-h v`
